lms/envs/minimal.yml

```diff
@@ -0,0 +1,22 @@
+# WARNING: Experimental
+#
+# This is the minimal settings you need to set to be able to get django to
+# load when using the production.py settings files. It's useful to point
+# LMS_CFG and CMS_CFG to this file to be able to run various paver commands
+# without needing a full docker setup.
```

Comment:
Can you link a followup ticket here, like [this one](https://github.com/openedx/wg-developer-experience)? I feel better about checking in experimental code when I know it'll be easy for someone in the future to check up on whether the code can be either polished or removed.

Reply:
I linked to the same ticket for now(https://github.com/openedx/wg-developer-experience/issues/136).